desktop/dev-environments/create-compose-dev-env.md

```diff
@@ -22,12 +22,12 @@ The example below, taken from the `compose-dev-env` project from the [Docker Sam
 
 >Note
 >
->If you want to create a Compose Dev Environment from a subdirectory of a Git repo, you need to define your own compose file in a .docker folder located in your subdirectory as currently, Dev Environments is not able to detect the main language of the subdirectory. 
+>If you want to create an advanced dev environment from a subdirectory of a Git repo, you need to define your own compose file in a .docker folder located in your subdirectory as currently, Dev Environments is not able to detect the main language of the subdirectory. 
```

Comment:
One last stray .docker snuck through!

Copying what we wrote in the other "Create" doc:

```suggestion
>Currently, Dev Environments is not able to detect the main language of the subdirectory. You need to define your own base image or compose services in a `compose-dev.yaml` located in your subdirectory.
```